core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9564,4 +9564,19 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withBigQuery()
         .ok(expectedBQSql);
   }
+  @Test public void testTimeOfDayFunction() {
+    final RelBuilder builder = relBuilder();
+    final RexNode formatTimestampRexNode2 = builder.call(SqlLibraryOperators.FORMAT_TIMESTAMP,
+        builder.literal("TIMEOFDAY"), builder.scan("EMP").field(4));
```

Comment:
instead of builder.scan("EMP").field(4) do 
builder.call(SqlLiberaryOperators.CURRENT_TIMESTAMP)